jsonnet/kube-prometheus/addons/windows.libsonnet

```diff
@@ -0,0 +1,57 @@
+local windowsdashboards = import 'kubernetes-mixin/dashboards/windows.libsonnet';
+local windowsrules = import 'kubernetes-mixin/rules/windows.libsonnet';
+
+{
+  values+:: {
+    windowsScrapeConfig+:: {
+      job_name: 'windows-exporter',
+      static_configs: [
+        {
+          targets: [error 'must provide targets array'],
+        },
+      ],
+    },
+
+    grafana+:: {
+      dashboards+:: windowsdashboards {
+        _config: $.kubernetesMixin.mixin._config {
+          wmiExporterSelector: 'job="' + $.values.windowsScrapeConfig.job_name + '"',
+        },
+      }.grafanaDashboards,
+    },
+  },
+  prometheus+: {
+    local p = self,
+    local sc = [$.values.windowsScrapeConfig],
+    prometheusRule+: {
```

Comment:
Since this is enabling a part of kubernetes-mixin, it would be better to include it in `kubernetesMixin.prometheusRule` - https://github.com/prometheus-operator/kube-prometheus/blob/master/jsonnet/kube-prometheus/main.libsonnet#L99-L102

Reply:
sounds good. I didn't quite know the difference between `promethues.mixin` and the `kubernetesMixin.prometheusRule`

Reply:
I've made this update. Makes more sense now thanks!